src/Surfnet/Stepup/Identity/Value/NameId.php

```diff
@@ -39,7 +39,7 @@ final class NameId implements JsonSerializable, Stringable
 
     public function __construct(string $value)
     {
-        if (strlen($value) > self::MAX_LENGTH) {
+        if (strlen($value) > self::MAX_LENGTH || strlen($value) === 0) {
             throw new InvalidArgumentException(
                 'Invalid argument type: maximum length for nameId exceeds configured length of ' . self::MAX_LENGTH,
             );
```

Comment:
Feels like the added condition should result in an updated exception message?

Current: `Invalid argument type: maximum length for nameId exceeds configured length of ' . self::MAX_LENGTH,`

Suggestion: `Invalid argument type: maximum length for nameId exceeds configured length of ' . self::MAX_LENGTH . ' the value can not be an empty string'`

Reply:
I splitted the logic in two exceptions.